lib/msal-electron-proof-of-concept/src/AppConfig/ServerRequest/ServerRequestParameters.ts

```diff
@@ -0,0 +1,28 @@
+import { Authority } from "../Authority/Authority";
+import { timingSafeEqual } from "crypto";
+
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+export class ServerRequestParameters {
```

Comment:
does this actually create a server request url?

Reply:
It was going to but not anymore. I’m changing it to an abstract class. Still a WIP right now. I’m going to use this as the base class for both Authorization and Token request parameters (both request URLs are different, but have stuff in common)

Reply:
So the inherited class is yet to come?